recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
   sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
-    - python.app  # [osx]
```

Comment:
Not sure. So far I haven't seen one yet. Do you know anything about it @ccordoba12?

@jjhelmus is our expert in packaging Python. Do you recommend anything? Is there an alternative?